protocol/java/org/openyolo/protocol/CredentialRetrieveRequest.java

```diff
@@ -130,10 +137,14 @@ public static CredentialRetrieveRequest fromRequestIntent(
     @NonNull
     private final Map<String, byte[]> mAdditionalProps;
 
+    @NonNull
+    private final Optional<Boolean> mRequireUserMediation;
```

Comment:
To avoid using type with a narrow support range, I'd just use a `Boolean` to get the trinary state you want. Alternatively, since the default will always be false, just use a primitive `boolean`.